{"uuid": "f36e7e74-248a-446a-99e1-7aadc0d80009", "children": ["c07e2028-a168-4929-a077-4ac63d42157b", "4d34f779-0add-4511-bd72-6e5572b2c3d9", "6fdb7e2d-1c9e-4eb7-988d-9669aede9888", "8e59b62c-35b5-4a80-97a7-3d329440a424", "7bf0712b-5445-46e5-be4c-6e925c4862e7"], "befores": [{"name": "_Class__pytest_setup_class", "status": "passed", "start": 1586835795373, "stop": 1586835795373}], "afters": [{"name": "_Class__pytest_setup_class::0", "status": "broken", "statusDetails": {"message": "NameError: name 'mail_pass' is not defined\n", "trace": "  File \"/usr/local/lib/python3.7/site-packages/allure_commons/_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/fixtures.py\", line 800, in _teardown_yield_fixture\n    next(it)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/python.py\", line 707, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/python.py\", line 634, in _call_with_optional_argument\n    func(arg)\n  File \"/Users/sonic/Project/Python/Pytest/scripts/test_case.py\", line 41, in teardown_class\n    EmailHandler().send_email()\n  File \"/Users/sonic/Project/Python/Pytest/uti/EmailHandler.py\", line 59, in send_email\n    smtp_obj.login(mail_user, mail_pass)\n"}, "start": 1586835795536, "stop": 1586835808183}], "start": 1586835795373, "stop": 1586835808184}